From d2a1288fba5d2a195c0fd3bfea43af72bea658b6 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:07:18 -0400
Subject: [usb] option driver, add support for Telit UE910v2

Message-id: <1410887463-20674-367-git-send-email-dzickus@redhat.com>
Patchwork-id: 95368
O-Subject: [RHEL7 PATCH 366/591] usb: option driver, add support for Telit UE910v2
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit d6de486bc22255779bd54b0fceb4c240962bf146
Author: Daniele Palmas <dnlplm@gmail.com>
Date:   Wed Apr 2 11:19:48 2014 +0200

    usb: option driver, add support for Telit UE910v2

    option driver, added VID/PID for Telit UE910v2 modem

    Signed-off-by: Daniele Palmas <dnlplm@gmail.com>
    Cc: stable <stable@vger.kernel.org>
    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/option.c b/drivers/usb/serial/option.c
index 68fc9fe..367c7f0 100644
--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -243,6 +243,7 @@ static void option_instat_callback(struct urb *urb);
 #define TELIT_PRODUCT_CC864_DUAL  0x1005
 #define TELIT_PRODUCT_CC864_SINGLE  0x1006
 #define TELIT_PRODUCT_DE910_DUAL  0x1010
+#define TELIT_PRODUCT_UE910_V2   0x1012
 #define TELIT_PRODUCT_LE920   0x1200
 
 /* ZTE PRODUCTS */
@@ -1041,6 +1042,7 @@ static const struct usb_device_id option_ids[] = {
  { USB_DEVICE(TELIT_VENDOR_ID, TELIT_PRODUCT_CC864_DUAL) },
  { USB_DEVICE(TELIT_VENDOR_ID, TELIT_PRODUCT_CC864_SINGLE) },
  { USB_DEVICE(TELIT_VENDOR_ID, TELIT_PRODUCT_DE910_DUAL) },
+ { USB_DEVICE(TELIT_VENDOR_ID, TELIT_PRODUCT_UE910_V2) },
  { USB_DEVICE(TELIT_VENDOR_ID, TELIT_PRODUCT_LE920),
   .driver_info = (kernel_ulong_t)&telit_le920_blacklist },
  { USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, ZTE_PRODUCT_MF622, 0xff, 0xff, 0xff) }, /* ZTE WCDMA products */
-- 
1.7.1